src/index.handlebars

```diff
@@ -172,14 +172,15 @@
                </div>
             </div>
             <div class="col-xs-12 col-sm-offset-3 col-sm-6 col-md-offset-0 col-md-4">
-                <h2 class="text-center">Our REST API reference</h2>
+                <h2 class="text-center rest">Our references</h2>
```

Comment:
Why do you add a `rest` class here ?

Reply:
We now have 2 colors for the API references: one for the REST API and another one for the Events API.

Reply:
I forgot to delete it after some CSS tests.